src/components/button/FavButton.tsx

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable tailwindcss/no-custom-classname */
+export const FavButton = () => {
+  return <button className="text-sm rounded-lg bg-tomato px-3 py-1 text-white"> お気に入りに追加 </button>;
+};
```

Comment:
こんな感じで実装すればフォローボタンを吸収してボタンコンポにできるかと思います。
色周りは適当です。

```suggestion

import cc from "classcat";

type ButtonProps = {
  children: React.ReactNode;
  // ボタンの色
  color?: "tomato" | "white" | "gray";
  // クライアントコンポーネントになると思うのでその内下を実装
  // clickEvent
  // onClick?: () => void;
};

export const Button: React.FC<ButtonProps> = (props) => {
  const buttonColor = cc([
    "rounded-lg bg-tomato px-3 py-1 text-medium text-white",
    {
      "bg-gray": props.color === "gray",
    }
  ]);
  return <button className="">{props.children}</button>;
};

```